src/MultiSelect/index.js

```diff
@@ -149,8 +176,22 @@ class MultiSelect extends Component<Props, State> {
     }
   };
 
+  checkValues = () => {
+    const { inputValue, fullValue } = this.state;
+    const allValues = [];
```

Comment:
Can we use `this.props.field.values` instead of creating this new `allValues` array?